cpp/README.md

```diff
@@ -1,15 +1,15 @@
 # Alluxio C++ API
 
 ### environment variables configuration
-- $JAVA_HOME must been set
-- Add jvm path "$JAVA_HOME/jre/lib/i386/server" or "$JAVA_HOME/jre/lib/amd64/server"
+- $JAVA_HOME must be set
+- Java 8 is needed
+- Add path "$JAVA_HOME/jre/lib/i386/server(32bit)" or "$JAVA_HOME/jre/lib/amd64/server(64bit)"
```

Comment:
nit: "$JAVA_HOME/jre/lib/i386/server" (32bit)
So it's clear that the parentheses are not part of the path